src/editor/components/modals/file-modal/body.jsx

```diff
@@ -2,6 +2,7 @@ import React from "react";
 import PropTypes from "prop-types";
 import styled from "react-emotion";
 
+import DropTarget from "../../../../shared/components/drop-target";
```

Comment:
I would hesitate to have this implemented in the PR too soon. The file modal area is being pretty heavily revamped / moved around a bit in the code base, and once this PR graduates out of "draft" it'll already require a rebase and possibly a refactor. Could you focus on the front page instead? I don't think the drag and drop work will get clobbered there.

Reply:
actually, maybe I would just focus on the storybook as a starting point